openapi.yaml

```diff
@@ -305,19 +334,37 @@ components:
       scheme: bearer
 
   schemas:
-    CouponCode:
+    OrderSpendResult:
       type: object
       required:
-        - coupon_code
+        - type
       properties:
-        coupon_code:
-          description: "a coupon code - appears in a result object of an Order"
-          type: string
-          example: aaaa-bbbb-cccc
         type:
           type: string
-          enum:
-            - coupon
+          enum: [coupon, payment]
+
+    CouponCodeResult:
+      allOf:
+        - $ref: '#/components/schemas/OrderSpendResult'
+        - type: object
+          required:
+            - coupon_code
+          properties:
+            coupon_code:
+              description: a coupon code - appears in a result object of an Order
+              type: string
+              example: aaaa-bbbb-cccc
+
+    JWTBodyConfirmPaymentResult:
+      allOf:
+        - $ref: '#/components/schemas/OrderSpendResult'
+        - type: object
+          required:
+            - jwt
```

Comment:
add doc explaining what this JWT contain

Reply:
It has 2 lines down